src/haz3lweb/UpdateAction.re

```diff
@@ -62,6 +62,7 @@ type t =
   | SetMeta(set_meta)
   | UpdateExplainThisModel(ExplainThisUpdate.update)
   | ExportPersistentData
+  | ExportScratchSlide // TODO Combine exports
```

Comment:
@disconcision not sure if you have any reason I shouldn't do this and add all the other exports?

https://github.com/hazelgrove/hazel/blame/2bfaef4a36a671f9cbc0177274d6d0341292e527/src/haz3lweb/view/ExerciseMode.re#L294 and such just bypass the action flow